### PR TITLE
Bump ApplicationVersion to 1.3.0.0

### PR DESCRIPTION
**Pull request by mattock**

### CODE REVIEW

**Comment by mattock** [comment]
Version 1.3.0.0 is released with openvpn 2.4_beta2. Is there some other place for the version number which would show up in "File properties" dialog in Windows? I don't think this has any effect there, but I could be wrong. I think we should start following the same procedure as for OpenVPN-GUI, that is, update version in sources and add two tags (e.g. 1.3.0.0 and openvpn-2.4_beta2). Thoughts?

**Comment by selvanair** [comment]
Yes, file properties show 0.0.0.0. I have no idea how to get the version into the exec. That said it would be nice to have version/build update and tags matching each release..

**Comment by mattock** [comment]
I'll look into getting the version number to File properties, and then force-push new stuff to this PR.

**Comment by mattock** [comment]
@selvanair : with the force-pushed changes the version numbers show up properly in _File Properties -> Details_. The only field I was unable to populate was "File description". A test build is here:

* https://build.openvpn.net/downloads/temp/OpenVpnServ2-pr7-2.exe

Right now the version number is defined in five different places. This should be improved at some point.
 
Setting the language field would have afaict required setting up satellite assemblies, i.e. localized AssemblyInfo files for each localization (e.g. _en-US_). I don't think that is worth the effort for openvpnserv2.

File: OpenVpnService.sln [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```
Comment (mattock) [inline OpenVpnService.sln]: It looks like MonoDevelop (which I use) and Visual Studio (used by someone else) are fighting each other here. I can revert this change if necessary.

**Comment by selvanair** [comment]
I too use mono, so wouldn't care about this visual studio version change..

**Comment by mattock** [comment]
@selvanair : Ok, so I take that as an ACK?

**Comment by selvanair** [comment]
OK, checked the executable. Looks good. We have to just remember to update the version number (1.3.0.0) in two places (in AssemblyVersion.cs and the .csproj file).

ACK.

**Pull request merged**